### ISSUE TITLE
After starting training the model,it doesn't work and only see "learning rate 0.0002000 -> 0.0002000"

### ISSUE THREAD

**Issue by DYJ1234567890** [junyanz/pytorch-CycleGAN-and-pix2pix#1275]
I try to start training the model but it doesn't work and stay in the same position for a long time,may you tell me the reason?
My GPU is RTX5000 16G batchsize 1

**Comment by DYJ1234567890** [comment]
![image](https://user-images.githubusercontent.com/49092156/115687073-64fd4180-a38c-11eb-8fa2-eaf7f1671881.png)

**Comment by korjmorj** [comment]
You have a UserWarning, check it! There is a little mistake in the code, you need to fork it and put lr_scheduler.step() AFTER optimizer.step()

**Comment by simon-eda** [comment]
can you please provide a description on where and how to put lr_scheduler.step() AFTER optimizer.step()?

many thanks

**Comment by ApoorvaSuresh** [comment]
@simon-eda hey, did you find out what has to be changed to make it work? Please let me know :) Thank you!

**Comment by simon-eda** [comment]
> 
> 
> @simon-eda hey, did you find out what has to be changed to make it work? Please let me know :) Thank you!

i switched to anaconda enviroment and the issues does not come up anymore

**Comment by Marco1402** [comment]
This issue persists for me, even with anaconda environment. How about you @ApoorvaSuresh ?

@simon-eda, would you be so kind to provide details about your environment and which package versions you used?

**Comment by junyanz** [comment]
We did provide an [environment.yml](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/environment.yml), which should work.
